src/main-client-v2.ts

```diff
@@ -3190,6 +3191,328 @@ export class MainClient extends BaseRestClient {
+    params: DeleteApiKeyBrokerSubAccountParams,
+  ): Promise<{}> {
+    return this.deletePrivate('sapi/v1/broker/subAccountApi', params);
+  }
+
+  getSubAccountBrokerIpRestriction(params: {
+    subAccountId: string;
+    subAccountApiKey: string;
+  }): Promise<{
+    subaccountId: string;
+    ipRestrict: boolean;
+    apikey: string;
+    ipList: string[];
+    updateTime: number;
+  }> {
+    return this.getPrivate(
+      'sapi/v1/broker/subAccountApi/ipRestriction',
+      params,
+    );
+  }
+
+  getApiKeyBrokerSubAccount(
+    params: GetApiKeyBrokerSubAccountParams,
+  ): Promise<ApiKeyBrokerSubAccount[]> {
+    return this.getPrivate('sapi/v1/broker/subAccountApi', params);
+  }
+
+  getBrokerInfo(): Promise<GetBrokerInfoResponse> {
+    return this.getPrivate('sapi/v1/broker/info');
+  }
+
+  /**
+   *
+   * EXCHANGE LINK - Account Endpoints
+   *
+   */
+
+  transferBrokerSubAccount(
+    params: TransferBrokerSubAccountParams,
+  ): Promise<TransferBrokerSubAccount> {
+    return this.postPrivate('sapi/v1/broker/transfer', params);
+  }
+
+  getBrokerSubAccountHistory(
+    params: GetBrokerSubAccountHistoryParams,
+  ): Promise<BrokerSubAccountHistory[]> {
+    return this.getPrivate('sapi/v1/broker/transfer', params);
+  }
+
+  submitBrokerSubFuturesTransfer(params: {
+    fromId?: string;
+    toId?: string;
+    futuresType: number; // 1: USDT Futures, 2: COIN Futures
+    asset: string;
+    amount: number;
+    clientTranId?: string; // The max length is 32 characters
+  }): Promise<{
+    success: boolean;
+    txnId: string;
+    clientTranId?: string;
+  }> {
+    return this.postPrivate('sapi/v1/broker/transfer/futures', params);
+  }
+
+  getSubAccountFuturesTransferHistory(params: {
+    subAccountId: string;
+    futuresType: number; // 1: USDT Futures, 2: COIN Futures
+    clientTranId?: string;
+    startTime?: number;
+    endTime?: number;
+    page?: number;
+    limit?: number;
+  }): Promise<any> {
+    return this.getPrivate('sapi/v1/broker/transfer/futures', params);
+  }
+
+  universalTransferBroker(
+    params: UniversalTransferBrokerParams,
+  ): Promise<BrokerSubAccount> {
+    return this.postPrivate('sapi/v1/broker/universalTransfer', params);
+  }
+
+  getUniversalTransferBroker(
+    params: GetUniversalTransferBrokerParams,
+  ): Promise<BrokerSubAccount> {
+    return this.getPrivate('sapi/v1/broker/universalTransfer', params);
+  }
+
+  /**
+   *
+   * EXCHANGE LINK - Fee Endpoints
+   *
+   */
+
+  // USD & Coin-M can be found under API getIncome() (find "API rebate" in results)
+  getBrokerSpotRebateHistory(days: 7 | 30, customerId?: string) {
+    if (days === 7) {
+      return this.getPrivate('sapi/v1/apiReferral/rebate/recentRecord', {
+        customerId,
+      });
+    }
+    if (days === 30) {
+      return this.getPrivate('sapi/v1/apiReferral/rebate/historicalRecord', {
+        customerId,
+      });
+    }
+  }
+
+  /**
+   * Broker Endpoints - only on old docs
+   */
```

Comment:
Here are marked broker endpoints that are no longer in new docs. All broker urls now are under sapi/vx/broker/smth 
, I dont see this apiReferral, only in old docs. So I marked them, until all old docs become deprecated, I guess we can keep it like that.